atlasdb-lock-api/src/main/java/com/palantir/atlasdb/transaction/api/LockAwareTransactionTask.java

```diff
@@ -18,5 +18,5 @@
 import com.palantir.lock.HeldLocksToken;
 
 public interface LockAwareTransactionTask<T, E extends Exception> {
-    T execute(Transaction t, Iterable<HeldLocksToken> heldLocks) throws E;
+    T execute(Transaction tx, Iterable<HeldLocksToken> heldLocks) throws E;
```

Comment:
`tx` is OK, `txn` might be preferred? I don't care enough to make you change it, though :)